common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -333,6 +334,28 @@ def selected_children(self):
 
         return self.selected
 
+    @XBlock.handler
+    def reset_selected_children(self, context):
+        """
+        This is a handler that deletes a user state of selected children to be displayed, 
```

Comment:
I don't think we're doing this at the moment. We would probably want to do something like iterating on `self.selected` blocks and calling their [reset_problem](https://github.com/edx/edx-platform/blob/f9619d6cad30aee254f0884cf04c9c0213d04e13/common/lib/xmodule/xmodule/capa_base.py#L1580-L1636) methods.